tests/fuzzers/bls12381/bls12381_fuzz.go

```diff
@@ -27,46 +27,37 @@ import (
 	"math/big"
 
 	"github.com/consensys/gnark-crypto/ecc"
-	gnark "github.com/consensys/gnark-crypto/ecc/bls12-381"
+	bls12381 "github.com/consensys/gnark-crypto/ecc/bls12-381"
```

Comment:
Not a blocker -- with two bls libraries, I think the previous import alias of `gnark` was less ambiguous

Reply:
Yeah agreed.